### ISSUE TITLE
midi controller slider16table cause segmentation violation (at least on windows).

### ISSUE THREAD

**Issue by mbfgit** [csound/csound#1983]
slider8table works fine, however slider16table crashes immediately.

The issue looks to be in Engine/entry1.c where slider16table, slider32table and slider64table are defined to use SLIDER8t structs instead of SLIDER16t, SLIDER32t, SLIDER64t.

Also in OOps/midiops3.c there's a hard-coded (j < 8) instead of (j < n) in SLIDER_I_TABLE_INIT macro.

SLIDERIF macro seems to have the same hard-coded (j < 8) issue.

**Comment by vlazzarini** [comment]
Right, thanks for the very thorough report, quick to fix when no investigation is needed. I was curious about this because it is code I never used although I moved it from one location to another in the sources. I thought I perhaps had messed up, but no, it looks like the bug was there since the code was brought into the codebase (around 2003?). Congrats, you are the first person to have ever tried to use it!

Fixed in #1984
